presto-main-base/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -2373,7 +2372,11 @@ public static OptionalInt getConcurrentLifespansPerNode(Session session)
             return OptionalInt.empty();
         }
         else {
-            checkArgument(result > 0, "Concurrent lifespans per node must be positive if set to non-zero");
+            if (result < 0) {
```

Comment:
this should be `if (result <= 0)` right ?

Reply:
Oh..the previous block checks == 

```
        if (result == 0) {
            return OptionalInt.empty();
        }
```

Reply:
oops missed that. sounds good then. Thanks !